### ISSUE TITLE
swipe support for touchscreens

### ISSUE THREAD

**Issue by Cdetviler** [aprsworld/wsWebDisplay#91]

**Comment by Cdetviler** [comment]
http://hammerjs.github.io/getting-started/

and 

https://jquerymobile.com/

both have the ability to make this possible. Jquery mobile is in alpha, however, and they do not recommend using their framework for production applications

**Comment by kb0thn** [comment]
Hammer seems like the ticket.

On 3/9/2016 2:40 PM, Charlie Detviler wrote:

> http://hammerjs.github.io/getting-started/
> 
> and
> 
> https://jquerymobile.com/
> 
> both have the ability to make this possible. Jquery mobile is in alpha,
> however, and they do not recommend using their framework for production
> applications
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/aprsworld/wsWebDisplay/issues/91#issuecomment-194497940.

## 

-James Jefferson Jarvis
APRS World, LLC
+1-507-454-2727
www.aprsworld.com

**Comment by Cdetviler** [comment]
I got swipe events working. I still need to tweak the settings to get them right. There are two problems right now:

1) it requires a certain element on the page to register the swipes. I chose the content container that holds all of the elements. Since other elements are on top of it at a higher z-index, the swipes are not recognized sometimes. I am sure there is a solution, I just didn’t get to it tonight.

2) Sometimes when a camera is being loaded at the same time as a swipe, it will create an indefinite loading camera on the next layout that loads.
